config/prow/build-cluster/add-secrets.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Knative Authors
```

Comment:
```suggestion
# Copyright 2020 The Knative Authors
```

Reply:
Also nit: flor cosmetic reason, add one blank line between line 1 and 2 ?

Reply:
done